src/workers/postUpdated.ts

```diff
@@ -75,76 +292,49 @@ const worker: Worker = {
           return;
         }
 
-        const creatorTwitter =
-          data?.extra?.creator_twitter === '' ||
-          data?.extra?.creator_twitter === '@'
-            ? null
-            : data?.extra?.creator_twitter;
-
-        const authorId = await findAuthor(entityManager, creatorTwitter);
-        const title = data?.title || databasePost.title;
-        const becomesVisible = !!title?.length;
-
-        const { allowedKeywords, mergedKeywords } = await mergeKeywords(
+        const { mergedKeywords, fixedData } = await fixData({
+          logger,
           entityManager,
-          data?.extra?.keywords,
-        );
+          data,
+        });
 
-        if (allowedKeywords.length > 5) {
-          logger.info(
+        // See if post id is not available
+        let combinedPost: Partial<ArticlePost>;
+        if (!post_id) {
+          // Handle creation of new post
+          try {
+            combinedPost = await createPost({
+              logger,
+              entityManager,
+              data: fixedData,
+              submission_id: data?.submission_id,
+            });
+          } catch (e) {
+            return;
```

Comment:
Why do we ignore all errors?

Reply:
Basically they are handled inside the function, I just needed a quick way to return if it failed.
We can do it based on result if you prefer?

Reply:
Yes please. For me seeing an empty `catch` usually signals a bad practice